src/natcap/invest/habitat_quality.py

```diff
@@ -64,128 +60,116 @@
-                "weight relative to the other threats.  Depending on the "
-                "type of habitat under review, certain threats may cause "
-                "greater degradation than other threats. "
-                "DECAY: A string value of either exponential or "
-                "linear representing the type of decay over space for "
-                "the threat. See the user's guide for valid values "
-                "for these columns. "
-                "BASE_PATH: optional. Required if baseline LULC input. "
-                "The THREAT raster filepath for the base scenario where the "
-                "filepath is relative to the THREAT CSV input. Entries can "
-                "be left empty if there is no baseline scenario or if using "
-                "the baseline LULC for rarity calculations only. "
-                "CUR_PATH: required. The THREAT raster filepath for the "
-                "current scenario where the filepath is relative to the "
-                "THREAT CSV input. "
-                "FUT_PATH: optional. Required if threat LULC input. The "
-                "THREAT raster filepath for the future scenario where the "
-                "filepath is relative to the THREAT CSV input. Entries can "
-                "be left empty if looking at current scenario only."
-                ),
-            "name": "Threats Data"
+                "Table mapping each threat of interest to its properties and "
+                "maps of its distribution. The raster columns give filepaths "
+                "to maps of the relative intensity of each threat, ranging "
+                "from 0 to 1. Paths are relative to the threats table path."),
+            "name": "Threats Table"
         },
         "access_vector_path": {
-            "validation_options": {
-                "required_fields": ["access"],
-                "projected": True,
-            },
             "type": "vector",
+            **utils.PROJECTED,
+            "fields": {"access": {"type": "ratio"}},
+            "geometries": utils.POLYGONS,
             "required": False,
             "about": (
-                "A GDAL-supported vector file.  The input contains data on "
-                "the relative protection that legal / institutional / social "
-                "/ physical barriers provide against threats.  The vector "
-                "file should contain polygons with a field ACCESS. "
-                "The ACCESS values should range from 0 - 1, where 1 "
-                "is fully accessible.  Any cells not covered by a polygon "
-                "will be set to 1."),
-            "name": "Accessibility to Threats (Vector) (Optional)"
+                "Map of the relative protection that legal, institutional, "
+                "social, and physical barriers provide against threats. Any "
+                "cells not covered by a polygon will be set to 1."),
+            "name": "Accessibility to Threats"
```

Comment:
So for something like this, we're relying more on the Users Guide to get specifics on inputs. Is that right? For instance I'm specifically thinking about the description:
>... where 1 is fully accessible.

Reply:
Good catch, added it to a nested about section in aecc86d2